### DISCUSSION TITLE
Instagram scraper

### DISCUSSION THREAD

**Discussion by mlkorra** [tattle-made/memebox#5] [General]
The following [scraper](https://github.com/chris-greening/instascrape)  requires a session Id,would it be feasible for the project on a longer run? Because,instagram keeps blocking the session id and we need to create a new session id everytime AFAIK

**Comment by dennyabrain** [comment]
I see. what are our other options?

**Reply by mlkorra** [reply]
yeah,I am exploring on that,will keep posted

**Comment by mlkorra** [comment]
also,do you have any test urls to start with, for reels and posts?

**Reply by dennyabrain** [reply]
Nothing in particular, lets use these 

https://www.instagram.com/reel/CV-DtiZAuhs/?utm_medium=copy_link

https://www.instagram.com/reel/CVKobtmgTST/?utm_medium=copy_link

https://www.instagram.com/reel/CVKyObeIW4m/?utm_medium=copy_link

https://www.instagram.com/p/CHmfQ9xlWI_/?utm_medium=copy_link

**Reply by mlkorra** [reply]
that helps

**Comment by mlkorra** [comment]
For now proceeding with the Session ID,as we are sending small no. of requests unlike the mass scraping

**Reply by mlkorra** [reply]
Currently,providing the input url inside the script itself, also it downloads the media locally
